subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -1098,6 +1098,7 @@ uint8_t ll_adv_enable(uint8_t enable)
 			}
 
 			slot_us += (BYTES2US(adv_size, phy) + EVENT_IFS_MAX_US
+				  + scan_req_us + 2 * EVENT_IFS_MAX_US + scan_rsp_us
```

Comment:
This is not required for `PDU_ADV_TYPE_DIRECT_IND`. Also, `rx_to_us` not required, as we are calculating max time reservation to ensure transmitted PDUs dont get aborted when another radio event get scheduled at end of the time reservation.

Reply:
OK, so rx_to_us is not needed at all?
How about rxtx_turn_us? Is this then also needed twice per scan_req/scan_rsp exchange?

Reply:
Hi - trying to understand this slot reservation timing (again):
It seems to me that neither adding rx_to_us nor rxtx_turn_us is correct? 
Should we in fact be using a contribution of MAX(IFS_MAX_US, rxtx_turn_us) between rx and tx?